src/beta_inc.jl

```diff
@@ -1001,7 +1004,7 @@ function _beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64=1-p)
     acu = exp10(iex)
 
     #iterate
-    while true
+    for i in 1:maxiter
```

Comment:
Maybe
```suggestion
    for _ in 1:maxiter
```
since we don't use `i`?